website/src/content/docs/reference/cli.mdx

```diff
@@ -80,7 +80,7 @@ Shows the Biome version information and quit
 - **`    --verbose`** &mdash;
   Print additional diagnostics, and some diagnostics show more information.
 - **`    --config-path`**=_`PATH`_ &mdash;
-  Set the filesystem path to the directory of the biome.json configuration file
+  Set the directory of the biome.json configuration file and disable default configuration file resolution.
```

Comment:
This phrase is auto generated from the rust source code.

You have to update this comment

https://github.com/biomejs/biome/blob/c5a98a72def0a1db072fc3face35cb30241608dd/crates/biome_cli/src/cli_options.rs#L24

Then we will update the website before the release.

Reply:
Hey, I indeed updated this comment you point to in #1844, and as per the contribution guide, this is the PR that updates the website:

> If your PR requires some update on the website (new features, breaking changes, etc.), you should create a new PR once the previous PR is successfully merged.
When adding new features, the documentation should be part of a new PR, which will be merged right before the release.

In the context of this PR, shall I revert any changes to `cli.mdx` and you guys will auto-generate it just before the next release?

Reply:
Oh nice, sorry I completely forgot